internal/pkg/grpcutil/logger_test.go

```diff
@@ -74,7 +74,7 @@ func TestPayloadUnaryServerInterceptor(t *testing.T) {
 		errCh <- server.Serve(listener)
 	}()
 
-	dial, err := grpc.Dial(listener.Addr().String(),
+	dial, err := grpc.Dial(listener.Addr().String(), //nolint:staticcheck
```

Comment:
I feel we should fix these

Reply:
dial can just be replaced with `NewClient`

Reply:
I'll do that in the separate PR.